SU-SWS/stanford_events
# D8CORE-3957: setting the events cards to be one per row at the med br…

# READY FOR REVIEW

# Summary
- Fix the event cards when it is a single item in a paragraph row. 
- Make sure the padding-right only appears on the cards when in  lg breakpoint. This comes from the events card css. 
- Verify the Events cards when multiples still align in a grid. 

# Review By (Date)
- ASAP

# Urgency
- High

# Steps to Test

1. Pull in this change.
2. And the Stanford Basic one. https://github.com/SU-SWS/stanford_basic/pull/221
3. and the Stanford Profile Helper one. https://github.com/SU-SWS/stanford_profile_helper/pull/87 
2. Clear caches.
3. Create content. Put event cards in one row.
4. Put event cards in another row with a different piece of content. 
5. View the page on responsive.
6. Verify the cards are full width with even padding on the sides and the cards have space below each one.
7. Verify that at the large breakpoint  the cards have the padding right.

# Affected Projects or Products
- Events
- SBT, SOE, SAA themes
- Responsive only.
- https://github.com/SU-SWS/stanford_basic/pull/221
- https://github.com/SU-SWS/stanford_events/pull/63
- https://github.com/SU-SWS/stanford_profile_helper/pull/87

# Associated Issues and/or People
- D8CORE-3957
- @katrialesser 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
